etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/orders/Order.java

```diff
@@ -11,4 +11,14 @@ public interface Order<T> {
     String getFhirResourceId();
 
     String getPatientId();
+
+    String getPlacerOrderNumber();
+
+    String getSendingApplicationDetails();
```

Comment:
Could we return a `MessageHdDataType` here instead of `String`?

Reply:
Yes. I agree with this approach. This is what all the methods with details will return.